blablabla

```diff
@@ -0,0 +1,28 @@
+
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Private Key' was found. <!-- 296fb943-585b-4f66-8dc1-182d7e1b6117 -->